packages/tools/generator-pie-component/src/app/templates/pie-skeleton.__stories__.ts

```diff
@@ -9,8 +9,7 @@ const keptReference = Pie<%= componentName %>;
 
```

Comment:
that's a good point. the placeholder was `component` initially but I had to change it as the testing directory had a folder named `component` so it will be replaced. I see we use `skeleton` in Fozzie so I updated it to match it. 

what do you think of renaming it to `placeholder`?

Reply:
That makes sense to me :) placeholder works!